pkg/api/encode_prom.go

```diff
@@ -17,11 +17,18 @@
 
 package api
 
+type PromTLSConf struct {
+	Enable   bool   `yaml:"enable,omitempty" json:"enable,omitempty" doc:"set to true to enable tls for the prometheus endpoint"`
+	CertFile string `yaml:"certFile,omitempty" json:"certFile,omitempty" doc:"path to the certificate file"`
```

Comment:
Nit: in other places we use "Path" rather than "File"
E.g.
```golang
	CACertPath         string `yaml:"caCertPath,omitempty" json:"caCertPath,omitempty" doc:"path to the CA certificate"`
```

Just for consistency, we should align here